DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadReceiveBinding.cs

```diff
@@ -64,7 +64,7 @@ private async Task ReceiveInternal(string modelCardId)
           modelCard.ProjectName,
           modelCard.ModelName,
           modelCard.SelectedVersionId,
-          cts,
+          cts.Token,
```

Comment:
I just spotted the POC: comment and am wondering if this should be an IReceiveOperation<TMeta>, would this help?

Reply:
I think this was related to us having to deconstruct the `ModelCard` information, so we could keep the receive operation "UI independent".

We may very well end up needing an interface for both send and receive but as of now, I think depending on the type directly is not too bad.

Reply:
Added as item in the checklist of https://spockle.atlassian.net/browse/CNX-9136 with links to this conversation